api/application.py

```diff
@@ -1,7 +1,8 @@
 import logging
 import os
 
-from core.api.database_connection_middleware import DatabaseConnectionMiddleware
+from core.api.database_connection_middleware import \
```

Comment:
single line only pls

Reply:
it was sort --sl that did that, should I still change

Reply:
ye, use line length = 1000